ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -1,16 +1,49 @@
-export function generateMetadata(): string {
+import { LoopPermissions } from './loop-permissions';
+
+function isPermissionUndefined(permission: string[]): boolean {
+  if (permission.length === 0 || 
+      permission === undefined) {
+    return true;
+  } 
+  return false;
+}
+
+function buildUrlPermissions(permissions: LoopPermissions): string {
+  if (isPermissionUndefined(permissions.urlPermissions)) {
```

Comment:
I'd like to know everyone's opinions on if we should throw an exception and break the webpack build whenever there are no permissions defined.

Reply:
Is it possible to have a loop with no permissions? Seems like it would be possible for _one kind_ of permission to be missing (no need to access remote urls for example), but it also seems like _one of them_ would have to be present or the loop wouldn't do anything.

Reply:
I think it may be best to actually **stop** webpack compilation (and return an exception to the Loop Author) when there are no permissions defined.